resources/scrape.py

```diff
@@ -1,20 +1,24 @@
+"""All the Scrape API endpoints"""
+
 from flask import Response, current_app
 from flask_restful import Resource
 from database.models import ScrapedAuthor
 from database.models import Quote
-from utils.scraping import scrapeQuotes
-from utils.utils import objectToJson
+from utils.scraping import scrape_quotes
+from utils.utils import object_to_json
 
 
 class ScrapeApi(Resource):
+    """API for Scraping"""
+
     def get(self):
         """Scrape and add data to the database
 
         Returns:
             Response: JSON object of message success
         """
         try:
-            quotes = scrapeQuotes("https://quotes.toscrape.com/")
+            quotes = scrape_quotes("https://quotes.toscrape.com/")
```

Comment:
Declare scrape link as a constant.
Suggestion: You can keep a separate file for all the constants across the application.